sql/src/test/java/io/crate/planner/DeletePlannerTest.java

```diff
@@ -84,9 +85,7 @@ public void testMultiDeletePlan() throws Exception {
 
     @Test
     public void testDeleteWhereVersionIsNullPredicate() throws Exception {
-        expectedException.expect(UnsupportedOperationException.class);
-        expectedException.expectMessage(
-            "Filtering \"_version\" in WHERE clause only works using the \"=\" operator, checking for a numeric value");
+        expectedException.expect(VersionInvalidException.class);
```

Comment:
expectedMessage missing, on purpose?

Reply:
yep I figured that since the exception class doesn't allow a custom message in the constructor the expectMessage part is redundant anyway. But I added it anyway so it's consistent everywhere.